book/06-github/1-github.asc

```diff
@@ -2,7 +2,7 @@
 == GitHub
 
 (((GitHub)))
-GitHub is the single largest host for Git repositories, and is the central point of collaboration for millions of developers and projects.
+GitHub is one of the largest host for Git repositories, and is the central point of collaboration for millions of developers and projects.
```

Comment:
This change is incorrect. As of right now, GitHub is still the largest single Git host. We only included a chapter on GitHub because the average reader will most likely be using it.